dev/pyproject.toml

```diff
@@ -44,6 +44,7 @@ dependencies = [
     "jsonschema>=4.19.1",
     'pendulum>=2.1.2,<4.0;python_version<"3.12"',
     'pendulum>=3.0.0,<4.0;python_version>="3.12"',
+    'pendulum>=3.1.0 ; python_version>="3.13"',
```

Comment:
Is there a reason not bumping pendulum in general?

Reply:
yep. Seems to work fine, no conflicts.